snap/pack/pack.go

```diff
@@ -122,7 +122,7 @@ func loadAndValidate(sourceDir string, yaml []byte) (*snap.Info, error) {
 		return nil, fmt.Errorf("cannot validate snap %q: %v", info.InstanceName(), err)
 	}
 
-	if err := snap.ValidateSnapContainer(container, info, logger.Noticef); err != nil {
+	if err := snap.ValidateSnapContainer(container, info, logger.Debugf); err != nil {
```

Comment:
This seems little beyond the short term requirement. Concern is anyone relying on the log entries..., preference is to do this as part of larger rework for aggregation.